vlib/v/fmt/tests/array_init_keep.vv

```diff
@@ -15,4 +15,7 @@ fn main() {
 		*/
 		'eggs',
 	]
+	_ := []int{len: 10, cap: 10, init: 7}
+	_ := []map[string]string{len: 5, cap: 50, init: 'a'}
+	_ := []map[string][]int{len: 7, cap: 100, init: [1, 2]}
```

Comment:
The last 2 lines are not valid V code.

Reply:
Oops, you're right. I changed added the map afterwards. I will change it in a later PR.